test/persistence-hooks.suite.js

```diff
@@ -1131,8 +1133,10 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           done();
         });
       });
-
-      it('triggers `after save` hook on create', function(done) {
+      // TODO: [1] `this.isNewRecord()` in dao returns false and save was handleing
+      //  a new instance as well, but for `replaceById` instance needs to
+      //  exist, otherwise you get an error
```

Comment:
Yeah, I did not realise `prototype.save()` supports both new and existing instances. I think it's important to preserve that feature. Here is an example code to demonstrate how it could be used:

```js
var inst = new MyModel();
inst.name = 'Miroslav';
inst.save();
```